app/adapters/application.js

```diff
@@ -1,5 +1,29 @@
-import OsfAdapter from './osf-adapter';
+import DS from 'ember-data';
+export default DS.JSONAPIAdapter.extend({
+    host: 'http://localhost:8000',
+    buildURL(modelName, id, snapshot, requestType) {
```

Comment:
We really need to make this (localhost:8000) part of the config, otherwise it is going to be really hard to deploy.